docs/src/how_to/mult_vis_in_fig.md

```diff
@@ -17,9 +17,15 @@ uf_deconv = example_data("UnfoldLinearModelContinuousTime")
 uf = example_data("UnfoldLinearModel")
 results = coeftable(uf)
 uf_5chan = example_data("UnfoldLinearModelMultiChannel")
-d_singletrial, _ = UnfoldSim.predef_eeg(; return_epoched=true)
-times = -0.099609375:0.001953125:1.0
 data, positions = TopoPlots.example_data()
+dat, evts = UnfoldSim.predef_eeg(; 
```

Comment:
can we include this into https://github.com/unfoldtoolbox/UnfoldMakie.jl/blob/main/docs/example_data.jl ?

And is this different to the examples used there?

That would make it nicer imho :)